### DISCUSSION TITLE
SkyField usable as basis for realtime GoTo/follow system earth-moon?

### DISCUSSION THREAD

**Discussion by pe1cvj** [skyfielders/python-skyfield#836] [General]
Dear all,
I'm a Dutch radio amateur and pyton programmer, currently looking into options to realtime calculate the azitmuth and elevation from a specific point on earth towards the moon. This will be used as input for a realtime goto/follow system for controlling a big radio dish, used for  Earth-Moon-Earth (EME) radio experiments. I ran into SkyFields as possible option to do this  in Python.  Looking into detail,  I noticed SkyField has an impressive set of options. As I'm no astronomer  and not (yet) familiar with all astronomy semantics, I got a little overwhelmed  ;-). 
My question is: has anyone looked into such a set-up and can provide me with a simple example to get me started more quickly?
Thanks in advance,
Lex Peters

**Comment by davidmikolas** [comment]
Welcome PE1CVJ/Lex! Skyfield is easy and fun. You can create a topos object for your location and .observe(moon) to get az/el at any time or array of times you like. You could do that moment-by-moment, or just calculate an array at say 1 second intervals for the next month when the Moon is visible and above your lower elevation limit and just interpolate or lookup from it.

You'll get a distance as well, but in terms of an accurate one-way light time, we'll have to wait for others to comment.

What I don't know is if topos on the moon and the lunar libration stuff is working yet, I think others will add comments on that relatively soon as well.

The documentation is pretty easy to read a few starter examples: https://rhodesmill.org/skyfield/ and then more: https://rhodesmill.org/skyfield/api.html 

If you look around in Space Exploration Stack Exchange for the Skyfield tag https://space.stackexchange.com/questions/tagged/skyfield or in Astronomy SE https://astronomy.stackexchange.com/questions/tagged/skyfield you may find a lot of helpful examples as well, and you can always post a "How do I...?" question in one of those. I think Space SE might be more to your liking.

Here's one of my old questions, with a load of answers  https://space.stackexchange.com/questions/33359/hijacked-space-data-notable-instances-of-recovering-images-or-other-goodies-fro

and there's even an amateur-radio tag https://space.stackexchange.com/questions/tagged/amateur-radio

There is even a Ham SE as well! https://ham.stackexchange.com/

**Reply by pe1cvj** [reply]
Hi David,
Thank you for your quick reply and directions. I will dive further into this documentation and will experiment with the given examples. Many thanks so far, re. Lex
